src/docs/asciidoc/reference/manage/environment-intro.adoc

```diff
@@ -1,3 +1,10 @@
 :relative-path: ../../
 include::{docdir}/variables.adoc[]
 
+Your *application* may be deployed and executed in separate zones called *environments*. Each environment is a particular zone that you can organize as you wish. For example, you can have one environment based on your application lifecycle: `prod`, `pre-prod`, `dev`.
+
+.Environment for free trial
+[WARNING]
+====
+If you are using a free trial account, only one environment is available.
```

Comment:
I think we should add a contact us too in order to tell that this is by default but we can change that for our clients that would like to test more ZetaPush. Here it feels that it is not possible at all to have several environments